pkg/controllers/termination/terminate.go

```diff
@@ -57,28 +57,22 @@ func (t *Terminator) cordon(ctx context.Context, node *v1.Node) error {
 
 // drain evicts pods from the node and returns true when all pods are evicted
 func (t *Terminator) drain(ctx context.Context, node *v1.Node) (bool, error) {
-	// 1. Get pods on node
+	// Get evictable pods
 	pods, err := t.getPods(ctx, node)
 	if err != nil {
 		return false, fmt.Errorf("listing pods for node, %w", err)
 	}
-
-	// 2. Separate pods as non-critical and critical
-	// https://kubernetes.io/docs/concepts/architecture/nodes/#graceful-node-shutdown
+	// Skip node due to do-not-evict
 	for _, pod := range pods {
+		// https://kubernetes.io/docs/concepts/architecture/nodes/#graceful-node-shutdown
```

Comment:
I feel like this link is in a weird spot now without the mention of non-critical + critical pods, even though it was in a weird spot in the first place. Fits better above the drain function imo.

Reply:
Good point, moving above.